docarray/typing/tensor/abstract_tensor.py

```diff
@@ -225,6 +230,21 @@ def __docarray_validate_getitem__(cls, item: Any) -> Tuple[int]:
             raise TypeError(f'{item} is not a valid tensor shape.')
         return item
 
+    @classmethod
+    def __modify_schema__(cls, field_schema: Dict[str, Any]) -> None:
+        field_schema.update(type='array', items={'type': 'number'})
+        if cls.__docarray_target_shape__ is not None:
+            shape_info = (
+                '[' + ', '.join([str(s) for s in cls.__docarray_target_shape__]) + ']'
+            )
+            if reduce(mul, cls.__docarray_target_shape__, 1) <= 256:
+                # custom example only for 'small' shapes, otherwise it is too big to display
+                example_payload = orjson_dumps(np.zeros(cls.__docarray_target_shape__))
+                field_schema.update(example=example_payload)
+        else:
+            shape_info = 'not specified'
```

Comment:
```suggestion
            shape_info = 'any shape'
```

would not it be better like this ?

Reply:
hmm maybe? I feel like it might give a false sense of confidence though, that fact that no shape was specified doesn't automatically mean that any shape will work

Reply:
Okay, I see a parallel with python type hint:

```python
def f(x):
   ...
```
this is not specified

```python
def f(x: Any):
   ...
```

this means anything works.

Should we then have an Any shape stuff that would be different from note specified?

Reply:
> Should we then have an Any shape stuff that would be different from not specified?

For mypy Any and not specified are the same thing, so if we want to stay within this analogy that you are drawing then i would say no actually

Reply:
hmm okay fair

Reply:
then lets keep like this